blocks/bricks/conv.py

```diff
@@ -185,7 +185,7 @@ class ConvolutionalActivation(Sequence, Initializable):
         The application method to apply after convolution (i.e.
         the nonlinear activation function)
 
-    See :class:`Convolutional` for explanation of other parameters.
+    See: :class:`Convolutional`
```

Comment:
It's interpreted like that because there is no new section, so it thinks it's a parameter. Try the one I mentioned, it should work:
```
See Also
--------
:class:`Convolutional` for the other parameters
```